pymc3/tests/test_sampling.py

```diff
@@ -936,14 +933,14 @@ def test_ignores_observed(self):
         npt.assert_array_almost_equal(prior["positive_mu"], np.abs(prior["mu"]), decimal=4)
 
     def test_respects_shape(self):
-        for shape in (2, (2,), (10, 2), (10, 10)):
+        for shape in ((2,), (10, 2), (10, 10)):
             with pm.Model():
-                mu = pm.Gamma("mu", 3, 1, size=1)
-                goals = pm.Poisson("goals", mu, size=shape)
+                mu = pm.Gamma("mu", 3, 1)
+                assert mu.eval().shape == ()
```

Comment:
Why are you sampling this variable and then checking its shape?  What does that have to do with this test, which is apparently about the shapes of the subsequently produced traces?